client/express-checkout/index.js

```diff
@@ -487,6 +487,20 @@ jQuery( ( $ ) => {
 							} );
 					}, 250 )
 				);
+
+			// WooCommerce Deposits support.
+			// Trigger the "woocommerce_variation_has_changed" event when the deposit option is changed.
+			$(
+				'input[name=wc_deposit_option],input[name=wc_deposit_payment_plan]'
+			)
+				.off( 'change' )
```

Comment:
I was being extra cautious because the ECE element can be reinitialized, event handlers are attached each time it happens. To avoid attaching multiple handlers to the same selector, I remove any existing event handlers first. Although the elements are not being removed and added again, it seems unnecessary, but I prefer to keep it this way for safety. Does that sound good to you?

Reply:
I appreciate the caution, but if we're going to do that I think we should pass the specific handler we're removing to the function so we're not just removing every single event handler that may have been attached.

See the [off()](https://api.jquery.com/off/) docs for details.